docs/_scripts/distribution_repo_version.sh

```diff
@@ -1,14 +1,3 @@
-# Distributions are created asynchronously. Create one, and specify the publication that will
```

Comment:
Is there value in keeping these `.sh` files with only one command each? I was thinking of having them embedded in the docs directly and escapped as bash? wdyt?

Reply:
Like maybe we're testing with these somehow?

Reply:
I don't think that's a bad idea. I only kept them in the scripts because the CI runs them to ensure they work, but the CLI already has those tests, so it's probably redundant.

Reply:
I've seen projects test their command snippets by having a sphinx plugin handle it. I'm in favor of dropping all .sh file usage. @fao89 and @daviddavis wdyt?

Reply:
pulp_ansible is the first plugin to move docs to CLI, maybe we need to discuss it on the open floor?

Reply:
So the commands would live in the rst docs and then the plugin would extract + run them? Is that what you're proposing?

Reply:
Yes using doctest hopefully.